src/macros.rs

```diff
@@ -295,6 +317,22 @@ macro_rules! valkey_module {
                 return raw::Status::Err as c_int;
             }
 
+            // register filters on module load
+            $(
+                use paste::paste;
+                let mut cmd_filters_vec = Vec::new();
+                $(
+                    paste! {
+                        // creating a unique extern c filter function name:  __extern_c_ $filter_func
```

Comment:
Nice! Is this a way to allow extern c functions (with Rust callback functions embedded) to be provided dynamically into a Rust helper function?

One question I have about paste is - what does the function name look like? (Does it automatically include the ModuleName?)

It would be good to handle the case where two Modules have the same function names - as long as the module name is included, we should be good

Reply:
updated modules to be filter1 and filter2 with same filter names in each